src/main/java/de/tum/in/www1/artemis/web/rest/dto/LtiLaunchRequestDTO.java

```diff
@@ -10,6 +10,12 @@ public class LtiLaunchRequestDTO {
 
     private String lis_person_sourcedid;
 
+    private String lis_person_name_given;
```

Comment:
You might want to disable the method name checks (Codacy) for this file and add a comment why this specific format is needed.

Reply:
That's a good idea, I'll do that 👍